serverless/installing-openshift-serverless.adoc

```diff
@@ -21,26 +21,6 @@ For more information on using the MachineSet API, read about xref:../machine_man
 // Add or remove an instance of a machine in a MachineSet
 include::modules/machineset-manually-scaling.adoc[leveloffset=+2]
 
-
-// Installing Service Mesh
-[id="installing-service-mesh_{context}"]
-== Installing Service Mesh
-
-An installed version of Service Mesh is required for the installation of {ServerlessProductName}. For details, see the {product-title} documentation on xref:../service_mesh/service_mesh_install/installing-ossm.adoc[Installing Service Mesh].
-
-[NOTE]
-====
-Use the Service Mesh documentation for Operator installation only. Once you install the Operators, use the documentation below to install the Service Mesh Control Plane and Member Roll.
-====
-
-
-// Installing Service Mesh Control Plane
```

Comment:
For SRVKS-284 this and "Installing SMMR" need to stay intact. Only in SRVKS-285 can we remove all of these.